test/Altinn.Notifications.Tests/Notifications.Integrations/Profile/ProfileClientTests.cs

```diff
@@ -82,7 +88,39 @@ public async Task GetUserContactPoints_FailureResponse_ExceptionIsThrown()
         Assert.Equal(HttpStatusCode.ServiceUnavailable, exception.Response?.StatusCode);
     }
 
-    private Task<HttpResponseMessage> GetResponse(UserContactPointLookup lookup)
+    [Fact]
+    public async Task GetUserRegisteredContactPoints_SuccessResponse_NoMatches()
+    {
+        // Act
+        List<OrganizationContactPoints> actual = await _profileClient.GetUserRegisteredContactPoints(["12345678", "98754321"], "no-matches");
+
+        // Assert
+        Assert.Empty(actual);
+    }
+
+    [Fact]
+    public async Task GetUserRegisteredContactPoints_SuccessResponse_TwoListElementsReturned()
+    {
+        // Act
+        List<OrganizationContactPoints> actual = await _profileClient.GetUserRegisteredContactPoints(["12345678", "98754321"], "some-matches");
+
+        // Assert
+        Assert.Equal(2, actual.Count);
+        Assert.Contains(actual, ocp => ocp.OrganizationNumber == "123456789" && ocp.PartyId == 56789 && ocp.UserContactPoints.Any(u => u.UserId == 20001));
+    }
+
+    [Fact]
+    public async Task GetUserRegisteredContactPoints_FailureResponse_ExceptionIsThrown()
+    {
+        // Act
+        var exception = await Assert.ThrowsAsync<PlatformHttpException>(
+            async () => await _profileClient.GetUserRegisteredContactPoints(["12345678", "98754321"], "error-resource"));
+
+        Assert.StartsWith("ProfileClient.GetUserRegisteredContactPoints failed with status code", exception.Message);
+        Assert.Equal(HttpStatusCode.ServiceUnavailable, exception.Response?.StatusCode);
```

Comment:
The Response property is not nullable. I'm a little picky here, but the question mark is basically adding an unnecessary if check. (Might be that the complier detects that it's unnecessary and exclude it for us.)